src/chigame/api/tests/factories.py

```diff
@@ -106,3 +106,18 @@ class Meta:
         model = Chat
 
     tournament = SubFactory(TournamentFactory)
+
+
+class LobbyFactory(DjangoModelFactory):
+    class Meta:
+        model = Lobby
+
+    match_status = Lobby.Lobbied
+    name = Faker("word")
```

Comment:
We could use a sequence here from factory boy to denote different lobbies.

Reply:
Yes exactly thank you for catching this Gio